### ISSUE TITLE
hidden routes

### ISSUE THREAD

**Issue by yurii-github** [KABBOUCHI/laravel-ward#2]
you should make clear that it makes hidden routes available on front like /ward

ward.php : enable_routes => false

helps, but it's unclear unless you look into the code

regards

**Comment by yurii-github** [comment]
not sure how to do it on related packages but I'll try after I check your code tomorrow a bit more.

my current local fix was adding custom ward.php and disabling auto discover for ward package.

**Comment by yurii-github** [comment]
e-em.. for laravel-ward it's clear :/ it's unclear for nova-logs-tool because you are using there silently  laravel-ward

**Comment by KABBOUCHI** [comment]
yeah, i should add a note for ward config, but I mentioned the use of this lib it in the readme file.
